debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/MongoDbConnectorIT.java

```diff
@@ -92,7 +92,7 @@ public void shouldFailToValidateInvalidConfiguration() {
         Config result = connector.validate(config.asMap());
 
         assertConfigurationErrors(result, MongoDbConnectorConfig.HOSTS, 1);
-        assertConfigurationErrors(result, MongoDbConnectorConfig.LOGICAL_NAME, 1);
+        assertConfigurationErrors(result, MongoDbConnectorConfig.SERVER_NAME, 2);
```

Comment:
Why has the value changed?

Reply:
In the ConnectorConfig, we also validate that the server name is different from the history topic name. MongoDB did not previously validate the server name, so now that it does, it throws 2 errors instead. (This is also seen in the previous version of MySqlConnectorIT.java. In the original file the validation was performed and 2 errors are thrown.)